helm/kanister-operator/values.yaml

```diff
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 image:
-  repository: kanisterio/controller
+  repository: kanisterio/kanister-controller
```

Comment:
are we renaming the controller image?

Reply:
because then we have to change https://github.com/kanisterio/kanister/blob/0041a021da47c6526f3d34a087314cf255b8874f/helm/kanister-operator/values.yaml#L5 

and ALL other references

Reply:
There's a mismatch in the goreleaser file. I'll change that instead of this.